app/services/manage_donation.rb

```diff
@@ -1,4 +1,40 @@
-class ManageDonation < ManageAction
+class ManageDonation
+  def self.create(params)
+    new(params).create
+  end
+
+  def self.format_params(user_email:, user_country:, page_id:, amount:, exp_date:, card_num: '4111111111111111', additional_options = {})
+    page = Page.find(page_id)
+    result = {}
+    result[:donationpage] = {
+        page: [page.slug, 'donation'].join('-'),
```

Comment:
Do you think string interpolation in this instance would be easier to read? 
`page: "#{page.slug}-donation"`